financialaid/constants.py

```diff
@@ -47,7 +49,7 @@ class FinancialAidStatus:
         DOCS_SENT: "Documents Sent by User",
         PENDING_DOCS: "Started Applications",
         PENDING_MANUAL_APPROVAL: "Pending Approval (Documents Received)",
-        SKIPPED: "Skipped"
+        SKIPPED: "Skipped",
```

Comment:
The user can still see these applications by adding 'reset' to the URL, but feel free to leave it as is since this isn't exposing any vulnerability